app/src/androidTest/java/com/aletheiaware/perspective/android/GameActivityInstrumentedTest.java

```diff
@@ -258,12 +260,12 @@ public void screenshotWorld1Puzzle5() throws Exception {
     public void screenshotWorld1Puzzle6() throws Exception {
         captureScreenshot(PerspectiveAndroidUtils.WORLD_ONE, 6);
     }
-
+/*
     @Test
     public void screenshotWorld1Puzzle7() throws Exception {
         captureScreenshot(PerspectiveAndroidUtils.WORLD_ONE, 7);
     }
-
+*/
```

Comment:
How about using [@ignore](http://junit.sourceforge.net/javadoc/org/junit/Ignore.html) instead of commenting out the test?

Reply:
Good idea - that is a lot neater